src/Pages/Applications/DevtronApps/service.ts

```diff
@@ -18,10 +18,13 @@
 import { ResourceKindType, get } from '@devtron-labs/devtron-fe-common-lib'
 
 import { Routes } from '../../../config'
-import { AppConfigStatusResponse } from './service.types'
+import { AppConfigStatusResponse, EnvConfigResponse } from './service.types'
 import { DEFAULT_LANDING_STAGE } from './Details/AppConfigurations/appConfig.type'
 
 export const getAppConfigStatus = (appId: number, resourceKind?: ResourceKindType): Promise<AppConfigStatusResponse> =>
     get(
         `${Routes.APP_CONFIG_STATUS}?app-id=${appId}${resourceKind === ResourceKindType.job ? `&appType=${DEFAULT_LANDING_STAGE.JOB_VIEW}` : ''}`,
     )
+
+export const getEnvConfig = (appId: number, envId: number): Promise<EnvConfigResponse> =>
+    get(`${Routes.ENV_CONFIG}?appId=${appId}&envId=${envId}`)
```

Comment:
Please use getUrlWithSearchParams for the query params

Reply:
updated in next PR